content/es/loosely-coupled-architecture.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Arquitectura débilmente acoplada
```

Comment:
Por mencionar algunas opciones más:
* Arquitectura ligeramente acoplada
* Arquitectura [poco acoplada](https://docs.aws.amazon.com/es_es/prescriptive-guidance/latest/patterns/build-a-loosely-coupled-architecture-with-microservices-using-devops-practices-and-aws-cloud9.html)
* Arquitectura [débilmente acoplada](https://learn.microsoft.com/es-es/azure/architecture/framework/devops/principles#use-loosely-coupled-architecture).

Reply:
Se decidió utilizar **débilmente** acoplada